samples/vboxwrapper/vbox_common.h

```diff
@@ -18,8 +18,8 @@
 
 // Provide cross-platform interfaces for making changes to VirtualBox
 
-#ifndef BOINC_VBOX_COMMON_H
-#define BOINC_VBOX_COMMON_H
+#ifndef _VBOX_COMMON_H_
+#define _VBOX_COMMON_H_
```

Comment:
Identifiers that begin with underscore and uppercase letter are reserved for implementation. The code in master was just corrected.